test/fixtures/go-binaries/source/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# This shell script can be used to generate the binaries that are located in the parent folder.
+# Simply call this script with no arguments. Needs Docker to be installed.
+# The binaries here are used for the symbol parser and the Go version extraction test.
+
+set -ex
+
+versions="1.13.15 1.16.15 1.18.5 1.19.0"
```

Comment:
`latest` is not here

Reply:
Deliberately, I don't think we want to add `latest` binaries to the checked in files. Not having latest in here means we always build concrete versions, and if the regression-test should fail we can add the failing versions in here and have that built explicitly again.